src/pages/upgrades/index.tsx

```diff
@@ -19,193 +34,222 @@ import Translation from "../../components/Translation"
+  children: ReactNode[]
+}) => {
+  return (
+    <Box
+      as={InfoBanner}
+      ml={8}
+      m={{ base: "2rem 0", lg: undefined }}
+      {...props}
+    />
+  )
+}
 
-const Row = styled.div`
-  display: flex;
-  align-items: flex-start;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column;
-  }
-`
-
-const H2 = styled.h2`
-  text-align: left;
-  margin-top: 0;
-  margin-bottom: 1rem;
-`
-
-const CentreCard = styled(Card)`
-  flex: 1 1 30%;
-  min-width: 240px;
-  margin: 1rem;
-  padding: 1.5rem;
-  text-align: center;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex: 1 1 30%;
-  }
-`
-
-const StyledCardContainer = styled(CardContainer)`
-  margin-top: 2rem;
-  margin-bottom: 3rem;
-`
-
-const StyledCard = styled(Card)`
-  flex: 1 1 30%;
-  min-width: 240px;
-  box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
-  margin: 1rem;
-  padding: 1.5rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex: 1 1 30%;
-  }
-
-  &:hover {
-    border-radius: 4px;
-    box-shadow: 0px 8px 17px rgba(#000000, 0.15);
-    background: ${(props) => props.theme.colors.tableBackgroundHover};
-    transition: transform 0.1s;
-    transform: scale(1.02);
-  }
-`
-
-const Disclaimer = styled.div`
-  margin: 0rem 12rem;
-  display: flex;
-  text-align: center;
-  justify-content: center;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin: 0rem 2rem;
-  }
-`
-
-const StyledInfoBanner = styled(InfoBanner)`
-  margin-left: 2rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin: 2rem 0;
-  }
-`
-
-const Vision = styled.div`
-  margin-top: 4rem;
-`
-
-const ContributeCard = styled.div`
-  border-radius: 2px;
-  border: 1px solid ${(props) => props.theme.colors.border};
-  padding: 2rem;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin: 0rem auto;
-  margin-bottom: 2rem;
-  max-width: 100ch;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin-left: 0rem;
-    margin-right: 0rem;
-    flex-direction: column;
-    align-items: flex-start;
-  }
-`
-
-const StyledCallout = styled(CalloutBanner)`
-  margin-left: 0rem;
-  margin-right: 0rem;
-`
-
-const ContributeButton = styled(ButtonLink)`
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    width: 100%;
-    margin-top: 1.5rem;
-  }
-`
-
-const Staking = styled.div`
-  padding: 4rem;
-  background: ${(props) => props.theme.colors.cardGradient};
-  width: 100%;
-  margin-top: 2rem;
-  display: flex;
-  flex-direction: column;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    padding: 2rem;
-  }
-`
-
-const StakingColumns = styled.div`
-  display: flex;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    flex-direction: column;
-  }
-`
+const PageCard = (props) => (
+  <Box as={Card} flex="1 1 30%" minW="240px" m={4} p={6} {...props} />
+)
```

Comment:
We can render the `Card` directly, right? no need to use `as` here.
```suggestion
const PageCard = (props: ComponentPropsWithRef<typeof Card>) => (
  <Card flex="1 1 30%" minW="240px" m={4} p={6} {...props} />
)
```